experimental/await-ops.md

```diff
@@ -0,0 +1,22 @@
+# [Await.ops](https://github.com/tc39/proposal-await.ops)
+
+## AwaitExpression
+
+```js
+interface AwaitExpression <: Expression {
+    type: "AwaitExpression";
+    argument: Expression;
+    operation: "all" | "any" | "race" | "allSettled" | undefined
```

Comment:
```suggestion
    operation: Identifier | null
```
Whitespaces are now allowed in `await.ops`. I suggest we align with what we did in [MetaProperty](https://github.com/estree/estree/blob/master/es2015.md#metaproperty).